### PR TITLE
Implement reward fund for workers #1020

### PR DESCRIPTION
**Pull request by maslenitsa93**
#1020

### CODE REVIEW

File: libraries/chain/database.cpp [hunk 1]
```diff
@@ -2557,9 +2557,9 @@ namespace golos { namespace chain {
        /**
         *  At a start overall the network has an inflation rate of 15.15% of virtual golos per year.
         *  Each year the inflation rate is reduced by 0.42% and stops at 0.95% of virtual golos per year in 33 years.
-        *  66.67% of inflation is directed to content reward pool
-        *  26.67% of inflation is directed to vesting fund
-        *  6.66% of inflation is directed to witness pay
+        *  60.003% of inflation is directed to content reward pool
+        *  24.003% of inflation is directed to vesting fund
+        *  5.994% of inflation is directed to witness pay
```
Comment (zxcat) [inline libraries/chain/database.cpp]: actually, we have only 2 fractional digits in percent type. and sum gives 90%, please, describe missing 10% too

File: libraries/protocol/include/golos/protocol/config.hpp [hunk 2]
```diff
@@ -100,10 +100,11 @@
 #define STEEMIT_INFLATION_RATE_START_PERCENT    (1500) // Fixes block 0 to 15%
 #define STEEMIT_INFLATION_RATE_STOP_PERCENT     (95) // 0.95%
 #define STEEMIT_INFLATION_NARROWING_PERIOD      (250000) // Narrow 0.01% every 250k blocks
-#define STEEMIT_CONTENT_REWARD_PERCENT          (66.67*STEEMIT_1_PERCENT) //66% of inflation,
+#define STEEMIT_CONTENT_REWARD_PERCENT          (60.003*STEEMIT_1_PERCENT) //60% of inflation,
```
Comment (zxcat) [inline libraries/protocol/include/golos/protocol/config.hpp]: it's bad idea

Comment (maslenitsa93) [reply libraries/protocol/include/golos/protocol/config.hpp]: @zxcat Bad idea is 3 digits after dot?

Comment (zxcat) [reply libraries/protocol/include/golos/protocol/config.hpp]: @maslenitsa93, nop

**Pull request merged**